### ISSUE TITLE
Fix capitalization issue 

### ISSUE THREAD

**Issue by frankambrosio3** [CDCgov/MicrobeTrace#358]
Polygon labels decapitalize after first letter.

**Comment by billswitzer2** [comment]
Are these the key labels? If so, I hate when that happens. Bill

From: frankambrosio3 ***@***.***>
Sent: Monday, April 19, 2021 1:53 PM
To: CDCgov/MicrobeTrace ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [CDCgov/MicrobeTrace] Fix capitalization issue (#358)


Polygon labels decapitalize after first letter.

**Comment by mossy426-cdc** [comment]
Yes, for example.  If a label was HHS, it would appear as Hhs for labels (also values for the keys) shown over the polygon.
